larpandora/LArPandoraInterface/LArPandora.h

```diff
@@ -34,6 +34,7 @@ namespace lar_pandora {
     void produce(art::Event& evt);
 
   protected:
+    void SetPandoraEventInformation(art::Event& evt);
```

Comment:
Unless the event needs to be modified, I suggest altering the signature:

```suggestion
    void SetPandoraEventInformation(art::Event const& evt);
```